pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -90,7 +90,7 @@ func newReconciler(mgr manager.Manager, actuator actuator.Actuator, platType con
 	r := &ReconcileCredentialsRequest{
 		Client:       mgr.GetClient(),
 		Actuator:     actuator,
-		platformType: platType,
+		PlatformType: platType,
```

Comment:
can we undo the changes of platformType to PlatformType once ReconcileStaleCredentialsRequest doesn't embed ReconcileCredentialsRequest?